src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -2788,8 +2789,31 @@ class spell_pri_vampiric_touch : public AuraScript
     }
 };
 
+// 193063 - Protective Light (Aura)
+class spell_pri_protective_light : public AuraScript
+{
+    PrepareAuraScript(spell_pri_protective_light);
```

Comment:
PrepareAuraScript was deprecated some time ago, it is not needed anymore

Reply:
aight, didnt knew